MapleServer2/Commands/Game/InfoCommands.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+using MapleServer2.Commands.Core;
+using MapleServer2.Packets;
+using MapleServer2.Tools;
+using MapleServer2.Enums;
+
+namespace MapleServer2.Commands.Game
+{
+    public class InfoCommands : InGameCommand
+    {
+        public InfoCommands()
+        {
+            Aliases = new[]
+{
+                "commands"
+            };
+            Description = "Give informations about all commands";
+        }
+
+        public override void Execute(GameCommandTrigger trigger)
+        {
+            List<CommandBase> commandList = CommandManager.CommandsByAlias.Values.ToList();
+            StringBuilder stringBuilder = new();
+
+            foreach (CommandBase command in commandList)
+            {
+                foreach (string alias in command.Aliases)
+                {
+                    stringBuilder.Append($"<font color='#d68a11'>/{alias}</font>");
```

Comment:
Create a string helper function that takes in the inner string and color as parameters and automatically wraps it like this